.travis.yml

```diff
@@ -20,27 +20,51 @@ before_install:
   - conda update -q conda
   # Useful for debugging any issues with conda
   - conda info -a
+
+  # install MHC predictors
+  - git clone https://mhcbundle:$MHC_BUNDLE_PASS@github.com/openvax/netmhc-bundle.git
+  - export NETMHC_BUNDLE_HOME=$PWD/netmhc-bundle
+  - mkdir tmp
+  - export NETMHC_BUNDLE_TMPDIR=$PWD/tmp
+  - export PATH=$PATH:$NETMHC_BUNDLE_HOME/bin
+
+  # needed for MHCflurry predictor test
+  - export KERAS_BACKEND=theano
+  # we want to include vaxrank unit tests as a general integration test to make sure the stack
+  # isn't broken in an obvious way
+  - git clone https://github.com/openvax/vaxrank.git
 addons:
   apt:
     packages:
-    # install pandoc for use with pypandoc for converting the README
-    # from markdown to RST
-    - pandoc
+      # Needed for NetMHC
+      - tcsh
+      # install pandoc for use with pypandoc for converting the README
+      # from markdown to RST
+      - pandoc
+env:
+  global:
+    # MHC_BUNDLE_PASS
+    - secure: "B2bLOYhDD1mSH6nBmUVoyoVzO8CsZ/qeduoQ/gq1UqPXyBa5cj8x1hYeYEXMVGH04dq6p5FHKIZoZOOmgtKoemje5enSV8IL2lttASqNvzsiyvy6I4C3r5Ve2H9HMLneDxkarfVVVhG2ZJyO7Y/vGN8STZWX6N3FJfCJ1B5kV5Q="
```

Comment:
Just confirming that this was also generated using a new `travis encrypt` command?

Reply:
yep it was